.github/workflows/ros2-ci.yaml

```diff
@@ -9,37 +9,44 @@ on:
 jobs:
 
   build-ros2:
-    runs-on: ubuntu-latest
+    runs-on: [ubuntu-latest]
     strategy:
       fail-fast: false
       matrix:
-        ros-version: [foxy, galactic]
+        ros_distribution:
+          - foxy
+          - galactic
+          - humble
+          - rolling
         include:
-          - ros-version: foxy
-            image: ros:foxy
-          - ros-version: galactic
-            image: ros:galactic
+          # Foxy Fitzroy (June 2020 - May 2023)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-latest
+            ros_distribution: foxy
+            ros_version: 2
+          # Galactic Geochelone (May 2021 - November 2022)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-focal-ros-galactic-ros-base-latest
+            ros_distribution: galactic
+            ros_version: 2
+          # Humble Hawksbill (May 2022 - May 2027)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-humble-ros-base-latest
+            ros_distribution: humble
+            ros_version: 2
+          # Rolling Ridley  (June 2020 - Present)
+          - docker_image: rostooling/setup-ros-docker:ubuntu-jammy-ros-rolling-ros-base-latest
+            ros_distribution: rolling
+            ros_version: 2
     container:
-      image: ${{ matrix.image }}
+      image: ${{ matrix.docker_image }}
     steps:
-      - name: Checkout Repo
-        uses: actions/checkout@v2
-      - name: Create Workspace
-        run: |
-          mkdir -p src_tmp/tvm_vendor
-          mv `find -maxdepth 1 -not -name . -not -name src_tmp` src_tmp/tvm_vendor/
-          mv src_tmp/ src/
-      - name: Install Dependencies
-        run: |
-          bash -c 'source /opt/ros/$ROS_DISTRO/setup.bash; \
-          apt-get update && rosdep update; \
-          rosdep install --from-paths src --ignore-src -y'
-      - name: Build Workspace
-        run: |
-          bash -c 'source /opt/ros/$ROS_DISTRO/setup.bash; \
-          colcon build --event-handlers console_cohesion+'
-      - name: Run Tests
-        run: |
-          bash -c 'source /opt/ros/$ROS_DISTRO/setup.bash; \
-          colcon test --event-handlers console_cohesion+; \
-          colcon test-result --verbose'
+      - name: setup directories
+        run: mkdir -p ros_ws/src
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          path: ros_ws/src
+      - name: build and test
+        uses: ros-tooling/action-ros-ci@v0.2
```

Comment:
Why did you change the CI scripts drastically?
If there is no specific reason, generally, it's recommended to keep the diff minimum.

Reply:
@kenji-miyake 
I adopted your suggestion at https://github.com/autowarefoundation/tvm_vendor/pull/9/commits/f36f3f4ce3a5015f09dae20cf81ac2a48512c533.